error-analyzer/src/main/java/com/google/sps/servlets/helloInLog4j2.java

```diff
@@ -0,0 +1,27 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
```

Comment:
Add a brief javadoc explaining what this does.

Reply:
Also class name and file name in PascalCase

Reply:
Is HelloInLog4J2 better?

Reply:
yes